spec/transaction/transaction.yml

```diff
@@ -939,6 +839,12 @@ components:
     #       items:
     #         $ref: "#/components/schemas/link"
     #       type: array
+    itemIdUpdate:
+      description: >-
+        Item identifier (unique per collection. If it doesn't match the `featureId` in path
+        parameters, then the Item is renamed.
+      example: smr200-200-4-2019
+      type: string
```

Comment:
To be discussed whether we want/should/must support this case, same with renaming assets.

Reply:
I've asked Pasquale and Dave on Jira about this. I'll wait on their answer.

Reply:
I'll merge as is and if the rename needs to be removed we can create a new PR, we will anyway need to change the code.